### DISCUSSION TITLE
Unwrapping

### DISCUSSION THREAD

**Discussion by Abdallahbalushi** [gmtsar/gmtsar#847] [Q&A]
I got this error when I am trying to do this step before undertaking the unwrapping. How can it be fixed? 
![image](https://github.com/gmtsar/gmtsar/assets/150544398/dbbc5fd9-4f05-4a7d-98b5-7b17be382802)
![image](https://github.com/gmtsar/gmtsar/assets/150544398/d507bb2b-e190-41cb-a608-29dd4f113478)

**Comment by Xiaohua-Eric-Xu** [comment]
I think you may need to run it in your F2/intf_all directory.

**Reply by Abdallahbalushi** [reply]
Thanks Eric, I tried that and did work!
Any ideas on how to plot the mask_def.grd file?? And how can i know where is my study area exact min/max coords to do a region cut from the plot?

Thanks in advance, 
Abdallah

**Reply by Xiaohua-Eric-Xu** [reply]
I would use a handy tool called `ncview`.

**Reply by rofiaini** [reply]
Hi eric, the unwrapping process on my computer takes a very long time, if I want to unwrapping in my AOI  whether to do it in the merge folder or F1/F2/

**Reply by Xiaohua-Eric-Xu** [reply]
A couple of ways to speed things up. 1) use higher coherence threshold; 2) downsample your data or cut to your AOI (run snaphu[_interp].csh with the cut parameter at the end); 3) filter your data more by setting a larger filter_wavelength.
